README.md

```diff
@@ -38,24 +38,34 @@ Commit your code regularly and use descriptive messages. This helps both you (in
 
 Demonstrate your understanding of this Sprint's concepts by answering the following free-form questions. Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your Team Lead.
 
-- [ ] Mention two parts of Express that you learned about this week.
+- [x] Mention two parts of Express that you learned about this week.
 
-- [ ] Describe Middleware?
+Server side routing and middleware with Express
 
-- [ ] Describe a Resource?
+- [x] Describe Middleware?
 
-- [ ] What can the API return to help clients know if a request was successful?
+Middleware provide a way to extend the features provided by the Express framework.
 
-- [ ] How can we partition our application into sub-applications?
+- [x] Describe a Resource?
+
+A resource is an object with a type, associated data, relationships to other resources, and a set of methods that operate on it.
+
+- [x] What can the API return to help clients know if a request was successful?
+
+Error messages
```

Comment:
If a request is successful, you want to either send a status(2XX), json({message: "Success"}) or both.

Sending an error message on successful request would be unusual.